src/components/KonnectorSuccess.jsx

```diff
@@ -77,35 +69,95 @@ export class KonnectorSuccess extends Component {
           >
             {hasLinks && (
               <SuccessLinks>
-                {displayDriveUrl && <DriveLink folderId={trigger.message.folder_to_save} />}
-                {displayBanksUrl && <BanksLink banksUrl={banksUrl} />}
+                {relatedApps.map((x, i) =>
+                  x.successLink(this.props, this.context, i)
+                )}
               </SuccessLinks>
             )}
           </DescriptionContent>
 
           <SuccessFooter>
-            <Button
-              label={successButtonLabel || t('account.success.button')}
-              onClick={event => {
-                event.preventDefault()
-                onDone(account)
-              }}
-            />
+            {relatedApps.length > 0
+              ? relatedApps[0].footerLink(this.props, this.context)
```

Comment:
Since apps was an object, the order could differe here, couldn't it?
Maybe it could be better to have this FooterLink componant as a separated component to be easier to customize?